### DISCUSSION TITLE
taskbar completely black - uninstall not working

### DISCUSSION THREAD

**Discussion by Juanxgalarza** [valinet/ExplorerPatcher#1239] [General]
![image](https://user-images.githubusercontent.com/69209298/229326484-d74483e1-ddf4-494a-9913-77574c5c3659.png)

**Comment by Juanxgalarza** [comment]
![image](https://user-images.githubusercontent.com/69209298/229326624-4efb8f70-e735-44f3-bd88-b27eb5e82a23.png)
(right click on taskbar)

**Comment by Silver-Ink** [comment]
I just disabled "Enable advanced mitigations for correct rendering using classic theme" under the _Advanced_ tab, then restarted explorer and now it's gone.
